src/Application/Adapters/Laravel/Commands/InsightsCommand.php

```diff
@@ -12,17 +12,16 @@
 use NunoMaduro\PhpInsights\Domain\Container;
 use NunoMaduro\PhpInsights\Domain\Kernel;
 use NunoMaduro\PhpInsights\Domain\Reflection;
+use RuntimeException;
 use Symfony\Component\Console\Output\ConsoleOutputInterface;
 
 /**
  * @internal
  */
 final class InsightsCommand extends Command
 {
-    /** @var string */
     protected $name = 'insights';
```

Comment:
Is this false positive or blocked by parent?

Reply:
Blocked by parent 👍